tower-scripts/bin/cicd-control.sh

```diff
@@ -55,6 +60,16 @@ export OPERATION=$2
 shift 2
 ARGS="$@"
 
+################################################
+# CLUSTER LOG GATHERING
+# PLEASE DO NOT ADD ANY NEW OPERATIONS BEFORE HERE
+################################################
+if [ "${OPERATION}" == "logs" ]; then
+
+  # Gather the logs for the specified cluster
+  ./gather-logs.sh ${CLUSTERNAME}
+fi
```

Comment:
Looks like this will fall through and corrupt the tarball with extraneous stdout.